app/controllers/concerns/paginable.rb

```diff
@@ -48,10 +48,17 @@ def searchable?
     # text {String} - Represents the text for the searchable link
     # page {String | Fixnum } - Represents the page to request for a search term
     def paginable_search_link(text = _('link name'), page = 1)
+      @paginable_path_params['search'] = params[:search] if searchable?
       url = paginable_base_url(page)
-      url+= "?search=#{params[:search]}" if searchable?
       return link_to(text, url, 'data-remote': true)
     end
+    # Generates an HTML link with clear search functionality
+    # text {String} - Represents the text for the searchable link
+    # page {String | Fixnum } - Represents the page to request for a search term
+    def paginable_clear_url(page = 1)
```

Comment:
this introduces side effects, i.e. paginable_clear_url is ONLY removing search param if present but keeping any query param present in the latest URL request. For example, if the search results where sorted by any field this will be held still. Is this what we want? As a user I would expect to clear any action made after hitting "Clear search results"

Reply:
If I cleared search results I would expect my sort rules to remain in place. Maybe a question for @stephaniesimms and @sjDCC.